kotlinx-coroutines-core/native/src/Debug.kt

```diff
@@ -5,14 +5,12 @@
 package kotlinx.coroutines
 
 import kotlin.math.*
+import kotlin.native.*
 
 internal actual val DEBUG: Boolean = false
 
-internal actual val Any.hexAddress: String get() = abs(id().let { if (it == Int.MIN_VALUE) 0 else it }).toString(16)
+internal actual val Any.hexAddress: String get() = abs(identityHashCode().let { if (it == Int.MIN_VALUE) 0 else it }).toString(16)
```

Comment:
We have UInt now. We don't need `abs` anymore. Can do `identityHashCode().toUInt().toString(16)`